packages/base-types/src/models/version/canvasTemplate.ts

```diff
@@ -0,0 +1,5 @@
+export interface CanvasTemplate {
```

Comment:
Can we add an id field so for the endpoints we can select by id instead of name?